lib/assets/javascripts/_modules/search.js

```diff
@@ -72,6 +78,20 @@
         $searchInput.focus();
         hideResults();
       });
+
+      // Attach analytics events to search result clicks
+      if(window.ga) {
+        $searchResults.on('click', '.search-result__title a', function() {
+          var href = $(this).attr('href');
+          ga('send', {
```

Comment:
Yeah this would be useful. I think your best bet here is using the [event flow report](https://support.google.com/analytics/answer/2521316?hl=en). With a custom "users with search event" segment, you should be able to get a useful view.